src/apps/views/AppsList/AppsList.tsx

```diff
@@ -145,6 +152,46 @@ export const AppsList: React.FC<AppsListProps> = ({ params }) => {
       }
     }
   });
+  const [activateApp, activateAppResult] = useAppActivateMutation({
+    onCompleted: data => {
+      const errors = data?.appActivate?.errors;
+      if (errors?.length === 0) {
+        notify({
+          status: "success",
+          text: intl.formatMessage(messages.appActivated)
+        });
+        refetch();
+        closeModal();
+      } else {
+        errors.forEach(error =>
```

Comment:
I could be mistaken here, but don't we have a generic error notifier for these types of errors now? cc @Cloud11PL

Reply:
Yes, I think this is unnecessary - checkout `makeMutation.ts` it should cover most of the cases